pkg/overlay/overlay.go

```diff
@@ -18,6 +18,20 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// Overlay type holds various configuration options for overlay
+// MountWithOptions accepts following type so its easier to specify
+// more verbose configuration for overlay mount.
+type Options struct {
+	// The upper directory is normally writable layer in an overlay mount.
+	Upperdir string
+	// The workdir is used to prepare files as they are switched between the layers.
+	Workdir string
+	// Graph options relayed from podman, will be responsible for choosing mount program
+	Graphopts []string
+	// Mark if following overlay is read only
+	Readonly bool
+}
```

Comment:
Shouldn’t this be `GraphOpts` and `ReadOnly`? I’m not sure about `UpperDir`, `WorkDir`.